external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -61,15 +61,42 @@
 
     <Copy Condition="'$(UAPToolsFolder)'!=''"
           SourceFiles="@(RunnerFolderContents)" 
-          DestinationFolder="$(NETCoreAppTestRootPath)\Runner\%(RecursiveDir)"
+          DestinationFolder="$(TestHostRootPath)\Runner\%(RecursiveDir)"
           SkipUnchangedFiles="true"/>
 
     <Copy Condition="'$(UAPToolsFolder)'!=''"
           SourceFiles="@(LauncherFolderContents)" 
-          DestinationFolder="$(NETCoreAppTestRootPath)\Launcher\%(RecursiveDir)"
+          DestinationFolder="$(TestHostRootPath)\Launcher\%(RecursiveDir)"
           SkipUnchangedFiles="true"/>
   </Target>
 
+  <Target Name="VerifyMatchingProjectNVersions"
+          BeforeTargets="Build">
+    <Error Condition="'$(ProjectNTfsExpectedPrerelease)' != '$(ProjectNTfsTestILCExpectedPrerelease)'" Text="The versions of ProjectN Targeting pack and TestILC have gone out of sync. Please make sure both of the prerelease versions on dependencies.props match." />
```

Comment:
FYI @stephentoub : from now on, if the autoPR is updating the ProjectN targeting pack prerelease but it doesn't update the TestILC prerelease as well, the build will fail. This is because we require these two to stay in sync and this was the best way we could think of preventing them go out of sync. If an autoPR fails, soon enough a new one should be open that updates the other prerelease so they are both in sync again.